test/ui-test.ts

```diff
@@ -31,6 +63,7 @@ describe("Test Excel Custom Functions", function () {
     describe("Get test results for custom functions and validate results", function () {
         it("should get results from the taskpane application", async function () {
             this.timeout(0);
+            initializeDebugger();
```

Comment:
should we consider adding initializeDebugger to the UI tests for our other repos? Or is this only relevant to custom functions?

Reply:
Yeah, we can move to a common location. Where do u suggest?

Reply:
I would suggest office-addin-debugging or office-addin-test-helpers. However I think make that a separate work item to move it there so your current work isn't blocked while waiting on moving this to a shared location